pkgs/applications/audio/openutau/default.nix

```diff
@@ -19,8 +19,8 @@ buildDotnetModule rec {
     hash = "sha256-HE0KxPKU7tYZbYiCL8sm6I/NZiX0MJktt+5d6qB1A2E=";
   };
 
-  dotnet-sdk = dotnetCorePackages.sdk_7_0;
-  dotnet-runtime = dotnetCorePackages.runtime_7_0;
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.runtime_8_0;
```

Comment:
If you drop the patch below, you can remove this as `dotnet-sdk` and `dotnet-runtime` are set to .NET 6 in `buildDotnetModule` by default.